_redirects

```diff
@@ -38,6 +38,7 @@
 /api/rapids-cmake/ /api/rapids-cmake/stable/
 /api/rmm /api/rmm/stable/
 /api/rmm/ /api/rmm/stable/
+/api/rapidsmpf/ /api/rapidsmpf/stable/
```

Comment:
Let's follow the pattern established above, and include both "with slash" and "without slash."

Also could we please alphabetize this? Put it right below rapids-cmake and above rmm.

```suggestion
/api/rapidsmpf /api/rapidsmpf/stable/
/api/rapidsmpf/ /api/rapidsmpf/stable/
```